--- origsrc/xf86-video-dummy-0.3.5/src/Makefile.am	2012-01-01 00:14:57.000000000 -0600
+++ src/xf86-video-dummy-0.3.5/src/Makefile.am	2012-03-27 06:23:00.034046000 -0500
@@ -29,6 +29,7 @@ AM_CFLAGS = $(XORG_CFLAGS) $(PCIACCESS_C
 
 dummy_drv_la_LTLIBRARIES = dummy_drv.la
 dummy_drv_la_LDFLAGS = -module -avoid-version
+dummy_drv_la_LIBADD = $(XORG_LIBS)
 dummy_drv_ladir = @moduledir@/drivers
 
 dummy_drv_la_SOURCES = \
